android/assets/jsons/Civ V - Gods & Kings/Buildings.json

```diff
@@ -477,7 +477,7 @@
 		"happiness": 4,
 		"greatPersonPoints": {"Great Engineer": 1},
 		"isWonder": true,
-		"uniques": ["Golden Age length increased by [50]%"],
+		"uniques": ["[+50]% length of Golden Ages"],
```

Comment:
Of? For?

Reply:
As it's being re-worded anyway, maybe "+XX% Golden Age length" or "+XX Golden Age duration"?